examples/wifi/nodemcu/waziup/code

```diff
@@ -0,0 +1,117 @@
+#include <SPI.h>
+#include <ESP8266WiFi.h>
+#include <ArduinoHttpClient.h>
+
+char ssid[] = "*******";   // your network SSID (name)
+char pass[] = "***********";    // your network password (use for WPA, or use as key for WEP)
+
+// Déclaration path and port
+char server[] = "api.waziup.io";
+char resource[]   = "/api/v2/devices/Water/ultrasonic/Distance/distance";
```

Comment:
The resource should be like "/api/v2/devices/<ID of your device>/sensors/<ID of your sensor>/value".
You can see the help in the sensor page on the Dashboard (API button on the top right).
For example or my sensor: http://dashboard.waziup.io/devices/MyDevice89/sensors/TC
The resource for the POST will be:  "/api/v2/devices/MyDevice89/sensors/TC/value"